README-sdk.md

```diff
@@ -0,0 +1,73 @@
+<img src="website/static/operator_logo_sdk_color.svg" height="125px"></img>
+
+[![Build Status](https://travis-ci.com/operator-framework/operator-sdk.svg?branch=master)](https://travis-ci.com/operator-framework/operator-sdk)
+[![License](http://img.shields.io/:license-apache-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0.html)
+[![Go Report Card](https://goreportcard.com/badge/github.com/operator-framework/operator-sdk)](https://goreportcard.com/report/github.com/operator-framework/operator-sdk)
```

Comment:
Should not it be pointing out downstream as well?

Reply:
No, README-sdk-md is strictly a copy of the UPSTREAM. Left untouched, just moved it out of the way.  See https://github.com/openshift/ocp-release-operator-sdk/pull/102#issuecomment-779416485